repo2docker/buildpacks/base.py

```diff
@@ -51,24 +51,24 @@
 
 # Base package installs are not super interesting to users, so hide their outputs
 # If install fails for some reason, errors will still be printed
-RUN apt-get update -qq && \
-    apt-get install -qq --yes --no-install-recommends \
+RUN apt-get -qq update && \
+    apt-get -qq install --yes --no-install-recommends \
        {% for package in base_packages -%}
        {{ package }} \
        {% endfor -%}
-    && apt-get purge -qq && \
-    apt-get clean -qq && \
+    && apt-get -qq purge && \
+    apt-get -qq clean && \
     rm -rf /var/lib/apt/lists/*
 
 # This apt-get install is *not* quiet, so users see their packages being installed
 {% if packages -%}
-RUN apt-get update -qq && \
-    apt-get install --yes \
+RUN apt-get -qq update && \
+    apt-get -qq install --yes \
```

Comment:
The comment above says this is not quiet, but the change makes it quiet.

Reply:
I wanted packages installed with apt.txt to not be quiet, so I've moved the comment appropriately